src/database/volunteerEntrySchema.ts

```diff
@@ -9,7 +9,7 @@ export type IVolunteerEntry = {
   eventId: string;
   roles: string[];
   volunteerId: string;
-  answers: IFormAnswer[];
+  responses: IFormAnswer[];
```

Comment:
I think this is a correct fix to our schema, but next time just let us know and we'll make a separate commit straight to develop. Good practice to keep commits, PRs, branches, etc. about one thing only.